Library/Koala/KoalaTests.swift

```diff
@@ -614,4 +614,49 @@ final class KoalaTests: TestCase {
     XCTAssertEqual("Apple", client.properties.last?["manufacturer"] as? String)
     XCTAssertEqual("Apple", callBackProperties?["manufacturer"] as? String)
   }
+
+  func testTrackViewedAccount() {
+    let client = MockTrackingClient()
```

Comment:
Since `KoalaTests` inherits from `TestCase` class, I wonder if we need to initialize a client for every test instead of using its parent's `trackingClient`. 🤔

Reply:
I think because it's a class you'll likely find the `client.events` overlapping each other and assertions may fail.

Reply:
Looks like we can use `trackingClient` from TestCase and then re-use it in each test like:
```
let koala = Koala(client: self.trackingClient)
```
and it would still work (since the client is being instantiated for each test separately.

However I think for now we should be good to go with this solution which is consistent with the rest of the test suite in `KoalaTests` where we instantiate the client for each test. 

I'm happy to re-factor this but currently it represents 37 instances of us instantiating client in each test like `let client = MockTrackingClient()` which could add up to this PR and make the codebase even more inconsistent with other tests.